cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -213,21 +213,73 @@ def single_qubit_randomized_benchmarking(
         A RandomizedBenchMarkResult object that stores and plots the result.
     """
 
+    qubits = cast(Iterator['cirq.Qid'], (qubit,))
+    result = parallel_single_qubit_randomized_benchmarking(
+        sampler,
+        qubits,
+        use_xy_basis,
+        num_clifford_range=num_clifford_range,
+        num_circuits=num_circuits,
+        repetitions=repetitions,
+    )
+    return result[qubit]
+
+
+def parallel_single_qubit_randomized_benchmarking(
+    sampler: 'cirq.Sampler',
+    qubits: Iterator['cirq.Qid'],
+    use_xy_basis: bool = True,
+    *,
+    num_clifford_range: Sequence[int] = tuple(np.logspace(np.log10(5), 3, 5, dtype=int)),
+    num_circuits: int = 10,
+    repetitions: int = 600,
+) -> dict:
```

Comment:
nit: Add full type annotation:
```suggestion
) -> Mapping['cirq.Qid', 'RandomizedBenchmarkResult']:
```